setup/includes/upgrades/common/3.0-policy-description.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Updates descriptions for policy templates using lexicon keys instead plain text.
+ */
+
+use MODX\Revolution\modAccessPolicy;
+
+$policies = $modx->getIterator(
+    modAccessPolicy::class,
+    ['name:IN' => modAccessPolicy::getCorePolicies()]
+);
+
+foreach ($policies as $policy) {
+    $policy->set('description', sprintf(
+        'policy_%s_desc',
+        str_replace([',', ' '], ['', '_'], strtolower($policy->get('name')))
+    ));
```

Comment:
It is pretty safe, because now in the database we store names as-is. And in constants, we also stored such values. So, if you changed something in the database, the following query just will not find such entries because it does not equal the requested values.
```
$policies = $modx->getIterator(
    modAccessPolicy::class,
    ['name:IN' => modAccessPolicy::getCorePolicies()]
);
```